### ISSUE TITLE
Incorrect comment in code example

### ISSUE THREAD

**Issue by ruckustboom** [godotengine/godot-docs#1608]
The page titled "Scripting (continued)" has the following code example:

```gdscript
func _someaction():
    s.queue_free() # immediately removes the node from the scene and frees it
```

Unless I'm mistaken, the whole point of `Node.queue_free()` is ***not*** to immediately/free the node.

**Comment by bojidar-bg** [comment]
The comment should say that it immediately removes the node from the tree and *afterwards* frees it when processing the frame is over.

**Comment by FeralBytes** [comment]
This is already fixed in master. Fixed by Pull #1575. Please close.

**Comment by bojidar-bg** [comment]
Ah, right, my bad. Closing as already fixed.
